mne/io/brainvision/tests/data/test_segmentation_impedance.vhdr

```diff
@@ -0,0 +1,143 @@
+Brain Vision Data Exchange Header File Version 1.0
+; Data created by the Vision Recorder
+
+[Common Infos]
+Codepage=UTF-8
+DataFile=test.eeg
+MarkerFile=test.vmrk
+DataFormat=BINARY
+; Data orientation: MULTIPLEXED=ch1,pt1, ch2,pt1 ...
+DataOrientation=MULTIPLEXED
+NumberOfChannels=19
+; Sampling interval in microseconds
+SamplingInterval=1000
+SegmentationType=MARKERBASED
+SegmentDataPoints=1100
+Averaged=YES
+AveragedSegments=80
+
+[Binary Infos]
+BinaryFormat=IEEE_FLOAT_32
+
+[Channel Infos]
+; Each entry: Ch<Channel number>=<Name>,<Reference channel name>,
+; <Resolution in "Unit">,<Unit>, Future extensions..
+; Fields are delimited by commas, some fields might be omitted (empty).
+; Commas in channel names are coded as "\1".
+Ch1=Fp1,,1,µV
+Ch2=Fp2,,1,µV
+Ch3=F3,,1,µV
+Ch4=F4,,1,µV
+Ch5=C3,,1,µV
+Ch6=C4,,1,µV
+Ch7=P3,,1,µV
+Ch8=P4,,1,µV
+Ch9=O1,,1,µV
+Ch10=O2,,1,µV
+Ch11=F7,,1,µV
+Ch12=F8,,1,µV
+Ch13=T3,,1,µV
+Ch14=T4,,1,µV
+Ch15=T5,,1,µV
+Ch16=T6,,1,µV
+Ch17=Fz,,1,µV
+Ch18=Cz,,1,µV
+Ch19=Pz,,1,µV
+
+[Comment]
+
+A m p l i f i e r  S e t u p
+============================
+Number of channels: 19
+Sampling Rate [Hz]: 1000
+Sampling Interval [µS]: 1000
+
+Channels
+--------
+#     Name      Phys. Chn.    Resolution / Unit   Low Cutoff [s]   High Cutoff [Hz]   Notch [Hz]    Series Res. [kOhm] Gradient         Offset
```

Comment:
See comment in other PR, this should eventually be added to `mne-testing-data`

Reply:
for the sake of this PR you could add the impedances to one of the existing VHDR files in https://github.com/mne-tools/mne-python/tree/master/mne/io/brainvision/tests/data and then use that file for testing.